app/client/cypress/integration/home/glossary.toggle.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Homepage', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it(`Glossary button toggles slide content`, () => {
+    cy.findAllByText('Glossary')
+      .filter(':visible')
+      .scrollIntoView()
+      .click();
+    cy.findByPlaceholderText('Search for a term...').should('be.visible');
+  });
+
+  it(`Glossary close button closes slide content`, () => {
+    cy.findAllByText('Glossary')
+      .filter(':visible')
+      .scrollIntoView()
+      .click();
+    cy.findByText('×')
+      .scrollIntoView()
+      .click();
```

Comment:
Same comment as above.

```suggestion
    cy.findAllByText('Glossary')
      .filter('[data-disabled="false"]')
      .click();
    cy.findByText('×').click();
```